rootless.h

```diff
@@ -1,24 +1,22 @@
-#include <sys/syslimits.h>
-#include <unistd.h>
+// TODO: remove this file
```

Comment:
Do we need any of the commented code? it's tracked in the git history and using `THEOS_PACKAGE_INSTALL_PREFIX` directly is not helpful is the vast majority of cases.

Reply:
Nope, left them there for reference. Can remove them outright if desired or we can wait until we remove the file entirely in the future.